.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/googletest"]
```

Comment:
Can you add a section at the bottom of the README for contributing that mentions that to run the tests you'll need to initialize the submodules (and instructions for how to do that)?

Reply:
Yes, that makes sense.